client/state/wordads/status/schema.js

```diff
@@ -0,0 +1,19 @@
+export const wordadsUnsafeValues = [ false, 'mature', 'private', 'spam', 'other' ];
+export const wordadsStatusSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+$': {
+			type: 'object',
+			required: [ 'ID' ],
+			properties: {
+				ID: { type: 'integer' },
+				name: { type: 'string' },
+				URL: { type: 'string' },
+				approved: { type: 'boolean' },
+				active: { type: 'boolean' },
+				unsafe: wordadsUnsafeValues,
```

Comment:
You can test if the schema works, by refreshing the page. You'll see a validation error, if the schema doesn't match:
<img width="871" alt="screen shot 2016-08-12 at 12 44 34 pm" src="https://cloud.githubusercontent.com/assets/1270189/17635130/9f93fdba-608a-11e6-84bc-256ded4c0380.png">